File: Python/uri_1131.py
cont = 0
inter = 0
gremio = 0
empates = 0
x = 0


def main():
    global cont
    global inter
    global gremio
    global empates
    global x
    on_off = True
    while on_off:
        i, g = map(int, input().split(' '))
        cont += 1
        if i == g:
            empates += 1
        elif i > g:
            inter += 1
        else:
            gremio += 1
        print('Novo grenal (1-sim 2-nao)')
        x = int(input())
        if x == 2:
            on_off = False
        elif x == 1:
            on_off = True


main()
print('%d grenais' % cont)
print('Inter:%d' % inter)
print('Gremio:%d' % gremio)
print('Empates:%d' % empates)
if inter > gremio:
    print('Inter venceu mais')
elif inter < gremio:
    print('Gremio venceu mais')
else:
    print('Nao houve vencedor')

File: Go/uri_1131.go
package main

import (
	"fmt"
	"os"
)

var i, g, e, n = 0, 0, 0, 0

func main() {
	n += 1
	var inter, gremio, x = 0, 0, 0
	fmt.Scanf("%d%d", &inter, &gremio)
	if inter == gremio {
		e += 1
	} else if inter > gremio {
		i += 1
	} else {
		g += 1
	}

	fmt.Println("Novo grenal (1-sim 2-nao)")
	fmt.Scanf("%d", &x)
	if x == 2 {
		fmt.Printf("%d grenais\nInter:%d\nGremio:%d\nEmpates:%d\n", n, i, g, e)
		if g > i {
			fmt.Printf("Gremio venceu mais\n")
		} else if i > g {
			fmt.Printf("Inter venceu mais\n")
		} else {
			fmt.Printf("Nao houve vencedor\n")
		}
	} else if x == 1 {
		main()
	} else {
		os.Exit(0)
	}
}
